card-admin/Model/Source/Cctype.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+declare(strict_types=1);
+
+namespace Pronko\LiqPayCardAdmin\Model\Source;
+
+use Magento\Payment\Model\Source\Cctype as PaymentCctype;
+
+class Cctype extends PaymentCctype
+{
+    /**
+     * Return all supported credit card types
+     *
+     * @return string[]
+     */
+    public function getAllowedTypes()
+    {
+        return ['VI', 'MC', 'AE', 'DI', 'JCB', 'DN'];
```

Comment:
Please see cctypes in the LiqPayAdmin/etc/config.xml.
There are only 2 currencies supported.

Reply:
Fixed